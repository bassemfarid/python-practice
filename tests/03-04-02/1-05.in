999999
1

123456
654321
exit
