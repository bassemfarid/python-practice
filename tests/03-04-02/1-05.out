The sum is: 1000000
The sum is: 777777
